README.md

```diff
@@ -4,3 +4,116 @@
+
+##### 1.2.2.1 Usage/Examples
+A valid TOMAR is a JSON file that contains an array of mapping rules. 
+Each mapping rule is a JSON object that contains two properties: `mqttTopic` and `kafkaTopic`. The following is an example of a valid TOMAR:
+```json
+[
+  {
+    "mqttTopic": "building/{building}/room/{room}/#",
+    "kafkaTopic": "building_{building}_room_{room}"
+  },
+  {
+    "mqttTopic": "sensors/#",
+    "kafkaTopic": "sensor_others"
+  },
+  {
+    "mqttTopic": "sensors/+/data",
+    "kafkaTopic": "sensor_data"
+  }
+]
+```
+Let's go through each rule:
+
+1. MQTT Topic: "building/{building}/room/{room}/#" -> Kafka Topic: "building_{building}room{room}"
+    This rule maps MQTT topics of the form "building/{building}/room/{room}/#" to the Kafka topic "building_{building}room{room}". The wildcard "#" represents one or more levels in the topic hierarchy. For example, if the MQTT topic is "building/A/room/1/floor/2", it will be mapped to the Kafka topic "building_A_room_1_floor_2".
+
+2. MQTT Topic: "sensors/#" -> Kafka Topic: "sensor_others"
+
+    This rule maps any MQTT topic starting with "sensors/" followed by any number of levels in the hierarchy to the Kafka topic "sensor_others". The "#" wildcard matches any number of levels. For example, if the MQTT topic is "sensors/temperature/living-room", it will be mapped to the Kafka topic "sensor_others".
+
+3. MQTT Topic: "sensors/+/data" -> Kafka Topic: "sensor_data"
+
+    This rule maps MQTT topics of the form "sensors/+/data" to the Kafka topic "sensor_data". The "+" wildcard matches a single level in the topic hierarchy. For example, if the MQTT topic is "sensors/temperature/data", it will be mapped to the Kafka topic "sensor_data".
+
+These mapping rules demonstrate how different MQTT topics can be mapped to corresponding Kafka topics based on predefined patterns. The MQTT Bridge uses these rules to map MQTT topics to Kafka topics.
+The order in which the rules are defined is important. The MQTT Bridge will use the first rule that matches the MQTT topic. For example, if the MQTT topic is "sensors/temperature/data", 
+it will be mapped to the Kafka topic "sensor_data" because the third rule matches the MQTT topic. If the third rule was not defined, the MQTT Bridge would use the second rule to map the MQTT topic to the Kafka topic "sensor_others".
+
+### 2. MQTT Bridge Configuration
+The user can configure the MQTT Bridge using the `application.properties` file. This section describes the configuration properties that can be used to configure the MQTT Bridge.
+The MQTT bridge can be configured using the appropriate prefix. Example:
+- `bridge.` is the prefix used for general configuration of the Bridge.
+- `mqtt.` is the prefix used for MQTT configuration of the Bridge.
+- `kafka.` is the prefix used for Kafka configuration of the Bridge.
```

Comment:
SHould we mention that they are MQTT server configuration and Kafka producer configurations?

Reply:
@kyguy sorry, I did not understand. I think it is already mentioning, `mqtt.` is the prefix used for `MQTT configuration` of the Bridge.

Reply:
@kyguy any insights for this?

Reply:
Sorry I meant something like this
```suggestion
- `kafka.` is the prefix used for Kafka producer configurations of the Bridge.
```
if you think it is useful, if not, please ignore

Reply:
I guess it is already implied

Reply:
Well, I think we are using this prefix not only for Kafka producer configuration.

Reply:
We actually have just the producer in our MQTT bridge. What other kind of Kafka configuration are you referring to? If you are referring to something line `kafka.bootstrap.servers` this is anyway a configuration for the producer.
We don't have it under `kafka.producer.` just because you got the code from the HTTP bridge where we decided to have common config across consumer and producer in a `kafka.` parameters.
In this case we could decide to not having a Kafka common section and having `kafka.producer.bootstrap.servers` but tbh I would leave as it is. Who knows if far in the future we'll decide to add an MQTT consumer part as well?